README.md

```diff
@@ -50,6 +51,7 @@ require("lazy").setup({
 			-- trunkPath = "trunk",
 			-- lspArgs = {},
 			-- formatOnSave = true,
+      -- formatOnSaveTimeout = 10, -- seconds
```

Comment:
Indentation is off. See https://github.com/trunk-io/neovim-trunk/blob/c5ab798387186e47ec29380ea5fd141e4b45c85c/README.md